### PR TITLE
Converting App to tsx

### PR DESCRIPTION
**Pull request by dailymp**
File app.js converted to app.tsx

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,9 +14,9 @@
     "express-jwt-authz": "^1.0.0",
     "jwks-rsa": "^1.3.0",
     "npm-run-all": "^4.1.3",
-    "react-router-dom": "^4.3.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "react-router-dom": "^4.3.1",
```
Comment (DavielSA) [inline package.json]: Can you check why change versión of react-douter-dom and devDependencies?

File: src/App.tsx [hunk 2]
```diff
@@ -11,8 +11,15 @@ import Courses from "./Courses";
 import PrivateRoute from "./PrivateRoute";
 import AuthContext from "./AuthContext";
 
-class App extends Component {
-  constructor(props) {
+interface Props {
+  history: object;
```
Comment (DavielSA) [inline src/App.tsx]: why history is the type object?

File: package.json (hunk 1 above)
Comment (dailymp) [reply package.json]: This is because I need the typings and just installed the library

File: src/App.tsx (hunk 2 above)
Comment (dailymp) [reply src/App.tsx]: Because it is an object but I don't know exactly which type

**Pull request merged**